app/code/Magento/Catalog/Api/ProductRepositoryInterface.php

```diff
@@ -14,7 +14,7 @@
 interface ProductRepositoryInterface
 {
     /**
-     * Create product
+     * use POST to Create product and PUT to Update product
```

Comment:
I prefer to revert this change, as this interface used not only for web api, but also as API for external modules.

Reply:
Reverted